tests/utils.py

```diff
@@ -0,0 +1,60 @@
+from eth_account.signers.local import LocalAccount
+
+from gnosis.eth.contracts import get_multi_send_contract
+from gnosis.eth.tests.utils import deploy_erc20
+
+
+def send_tx(w3, tx, account: LocalAccount):
+    if "nonce" not in tx:
+        tx["nonce"] = w3.eth.get_transaction_count(
+            account.address, block_identifier="pending"
+        )
+    if "gasPrice" not in tx and "maxFeePerGas" not in tx:
+        tx["gasPrice"] = w3.eth.gas_price
+    if "gas" not in tx:
+        tx["gas"] = w3.eth.estimate_gas(tx)
+    else:
+        tx["gas"] *= 2
+    signed_tx = account.sign_transaction(dict(tx))
+    tx_hash = w3.eth.send_raw_transaction(bytes(signed_tx.rawTransaction))
+    w3.eth.wait_for_transaction_receipt(tx_hash)
+
+
+def deploy_multisend(w3, account: LocalAccount):
```

Comment:
There's a function for deploying `multisend` on `gnosis.safe.multi_send` called `deploy_contract`